epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -86,6 +94,28 @@ private void cancelAnimation(LottieAnimationView lottieView) {
   }
 
   @Override
+  public boolean canAnimateChanges(ColorHolder view) {
+    return false;
+  }
+
+  @Override
+  public Animator animateChanges(ColorHolder view) {
+    int currentColor = ((ColorDrawable) view.getBackground()).getColor();
```

Comment:
You're right that using the state change to trigger the Lottie animation is unnecessary, I just did it as an example of how `bind` can be used with the previous model to see the difference, and the lottie animation is kind of fun.

We can change it to something else, but since it is an example project we should do something to show how to use that method.